barretenberg/cpp/src/barretenberg/benchmark/goblin_bench/goblin.bench.cpp

```diff
@@ -1,144 +0,0 @@
-
-#include <benchmark/benchmark.h>
-
-#include "barretenberg/common/op_count_google_bench.hpp"
-#include "barretenberg/goblin/goblin.hpp"
-#include "barretenberg/goblin/mock_circuits.hpp"
-#include "barretenberg/stdlib_circuit_builders/ultra_circuit_builder.hpp"
-
-using namespace benchmark;
-using namespace bb;
-
-namespace {
-
-class GoblinBench : public benchmark::Fixture {
```

Comment:
why was this deleted?

Reply:
Were never going to use goblin this way so its just clutter. There are still tests that cover all relevant functionality